tools/cicd/cloudbuild.yaml

```diff
@@ -45,4 +45,4 @@ steps:
 images: ['us-docker.pkg.dev/$PROJECT_ID/tools/hatch']
 
 substitutions:
-  _PYTHON_VERSION: 3.9.18
\ No newline at end of file
+  _PYTHON_VERSION: 3.10.0
```

Comment:
Use `3.10.13` to match the cicd version: https://github.com/Kaggle/kaggleazure/blob/1477121a732deec37e38a74be10ee902de683ddc/configs/cicd/cloud_builds/main.tf#L54